tests/integration/conftest.py

```diff
@@ -18,3 +19,10 @@ def debug_env_name():
 @fixture
 def product_info():
     return 'pytester', __version__
+
+
+@fixture
+def serverless_env():
```

Comment:
Please move fixture to the relevant test module only, we can broaden the scope later too

Reply:
done